### ISSUE TITLE
Site Privacy: Coming Soon screen still hiding site although site is set to Public

### ISSUE THREAD

**Issue by tellisbethel** [Automattic/wp-calypso#53626]
### Steps to reproduce the behavior

Not aware of how to reproduce this issue.

#### What I expected to happen
Expected site to be fully accessible to visitors because it was set to Public

#### What actually happened
Site was launched and the privacy setting was set to Public, but the Coming Soon screen was still blocking access

### Context
I encountered this in this chat: 30070505-hcv. 
(Advised to create bug report here: p1623349790028000-slack-C3GP81E05)

#### Does this happen on simple or atomic sites or both?
This was an atomic site in this case.

#### Workaround 
Toggled the setting to Coming Soon and back to Public and this fixed the issue 
(Workaround discovered in this thread: p1620314572296500-slack-C3GP81E05)

#### Is there any console output or error text?
Not sure. Found workaround before I had a chance to check.

#### Level of impact (Does it block purchases? Does it affect more than just one site?)
It blocks user's visitors' access to the site. 
This GH comment about another site looks to be referencing a similar issue: https://github.com/Automattic/wp-calypso/issues/48316#issuecomment-823406850

**Comment by nagpai** [comment]
I can confirm this commonly happening on Atomic sites, and get about one or two cases every two days. 

Recent one - #29879379-hc 

Solution for now: 
Toggling the site back to Coming soon, and then making it Public fixes it.

**Comment by nagpai** [comment]
Another issue, this time on simple site 
#30188088-hc

**Comment by nagrampai** [comment]
Another issue 
#10345607-hc

**Comment by nagpai** [comment]
Another issue 
10345607-hc

**Comment by poojashetty18** [comment]
#4091221-zen

**Comment by nagpai** [comment]
#30397850-hc
Simple site.

**Comment by tellisbethel** [comment]
Another instance here: 30595198-hc
Atomic site

**Comment by xpurichan** [comment]
30951205-hc
Atomic site

**Comment by poojashetty18** [comment]
#4208252-zen
Atomic site

**Comment by hacchism** [comment]
Another case on #4237346-zen
Simple site

**Comment by synora** [comment]
Simple site report at #4238213-zen

**Comment by metabreakr** [comment]
28585962-hc
Atomic

**Comment by rossanafmenezes** [comment]
4252510-zen (Simple site)

**Comment by poeticabdul** [comment]
I encountered this during an empathy flow testing using the simple site `empathyabdul.blog`. The workaround worked fine.

**Comment by lizswafford** [comment]
29596428-hc
Atomic

**Comment by ArSn** [comment]
31704338-hc
Atomic

**Comment by erricgunawan** [comment]
4309610-zen
Simple site

**Comment by jamiepalatnik** [comment]
Saw this again on another Atomic site, 31385110-hc

**Comment by hacchism** [comment]
31476637-hc
Simple site

**Comment by erricgunawan** [comment]
4357395-zen 
Simple site

**Comment by Nic-Sevic** [comment]
another here on AT site: 32111084-hc

**Comment by jamiepalatnik** [comment]
Another report: 17734307-hc

**Comment by as-esu** [comment]
4399055-zen ( Simple site )

**Comment by pauloeaquino** [comment]
4433317-zen
Atomic

**Comment by cuemarie** [comment]
Simple Site: 30640684-hc
Toggling to Coming Soon > Public resolved the issue

**Comment by jamiepalatnik** [comment]
More reports: 4437827-zd-woothemes
25151463-hc
4438149-zd-woothemes
32953077-hc

Resaved settings as a workaround in these cases.

**Comment by pauloeaquino** [comment]
4446511-zen (simple)

**Comment by as-esu** [comment]
4463485-zen ( simple )

**Comment by isocialtish** [comment]
Another report in 30192345-hc I resaved their privacy settings and it's resolved now

**Comment by poojashetty18** [comment]
4535086-zen ( Simple site )

**Comment by ariel-maidana** [comment]
Another report in 32893422-hc
It was solved by setting the site to Private and then to Public again.

**Comment by poeticabdul** [comment]
Another report in 4547559-zen
Solved by setting the site to Private and then to Public again

**Comment by formosattic** [comment]
32104112-hc
34908345-hc

Solved by setting the site to Private and then to Public again

**Comment by cuemarie** [comment]
+1 31596917-hc
Resolved by setting site to Coming Soon (and saving), then setting as Public again

**Comment by metabreakr** [comment]
32915458-hc

**Comment by hacchism** [comment]
Another instance: 4568840-zen

**Comment by poojashetty18** [comment]
4568840-zen (Simple site)

**Comment by gabriel-fuentes** [comment]
4576782-zen

**Comment by allilevine** [comment]
4600039-zen

**Comment by brookewp** [comment]
4601296-zen

**Comment by poojashetty18** [comment]
4616795-zen

**Comment by hacchism** [comment]
4628432-zen (Simple site)

**Comment by cuemarie** [comment]
4638090-zd-woothemes
Directed to resave settings

**Comment by brinbliss** [comment]
29128948-HC
Re-set to Coming Soon, saving, and then back to Public.

**Comment by essleeung** [comment]
Another one here- 32219231-hc
Did the Public- Coming soon- Public settings save to fix it.

**Comment by nagpai** [comment]
#33077597-hc

Turn it on and turn it off... worked.

**Comment by as-esu** [comment]
4686954-zen

**Comment by allilevine** [comment]
I haven't been able to reproduce this bug on a test site so far, but my guess is that it has something to do with our Launch flows. We handle launching a site differently from changing a site from "Coming Soon" to "Public" in Settings. Users can launch a site from either

Settings > General:
<img width="731" alt="Screen Shot 2022-01-17 at 11 58 59 AM" src="https://user-images.githubusercontent.com/1689238/149811526-07536d56-14a1-4cda-84c3-5b7109110235.png">

or My Home > Launch your site to the world:
<img width="704" alt="Screen Shot 2022-01-17 at 11 59 22 AM" src="https://user-images.githubusercontent.com/1689238/149811519-1e4dc3cc-e0ca-4bb8-b8dd-bc9fe2851551.png">

Currently the My Home launch flow [checks if the site has a paid plan and a domain](https://github.com/Automattic/wp-calypso/blob/7655b6d1f326371990851da10ebb97f133fbdc17/client/state/sites/launch/actions.js), OR is an Anchor podcast site. Otherwise, the flow takes the user to `/start/launch-site` and launches the site from there. 

It looks like the affected sites were all launched before the user asked for help. I'm not seeing any consistent data about the type of site, type of plan, or whether they had a domain before.

**Comment by hacchism** [comment]
#33525394-hc

Did the Public- Private- Public settings save to fix it.

**Comment by cuemarie** [comment]
Atomic Site: 33630997-hc
Switched to Coming Soon -> Public and that resolved

**Comment by retnonindya** [comment]
4714924-zd-woothemes

Doing the usual site privacy switcheroo helped

**Comment by nagpai** [comment]
#32946607-hc

**Comment by allilevine** [comment]
@Automattic/quality-squad I think this would be a great issue to prioritize because of the number of HE reports it's generating. It does have a workaround, however.

**Comment by rosepajaroja** [comment]
Similar issue reported here - hc-33883002 (AT)

I am not sure if this is a one-time thing, but this workaround fixes it: 

1 - Switch site's privacy from Public to Coming Soon. 
2 - Change primary domain name of the site to wpcomstaging.com. 
3 - Switch site's privacy from Coming Soon to Public. (Should work)
4 - Change primary domain name back to custom domain name.

**Comment by nagpai** [comment]
#4727225-zen

doing the privacy settings somersault helped like in most cases.

**Comment by essleeung** [comment]
Another one here: 4735122-zd-woothemes
Privacy somersault.

**Comment by brinbliss** [comment]
Two more today, both fixed by turning it back to Coming Soon and then Public again:
4725606-ZD
33982995-HC

**Comment by kylemcph** [comment]
Another one here: 4743468-zd-woothemes
Privacy somersault worked.

**Comment by tellisbethel** [comment]
Another report here: 4745702-zd-woothemes
"Privacy somersault" worked. <- Tee-hee! I like that lingo.

**Comment by kylemcph** [comment]
Another one here: #24050307-hc
Privacy somersault worked.

**Comment by brinbliss** [comment]
4764641-ZD 

🤸 worked!

**Comment by hacchism** [comment]
Another report - #34079829-hc
Privacy somersault worked.

**Comment by kylemcph** [comment]
Another report - #34171530-hc
Privacy somersault worked.

**Comment by nagpai** [comment]
34190531-hc
Privacy somersault worked

**Comment by csabarakasz** [comment]
Same issue, simple site: 4753759-zd-woothemes

**Comment by cuemarie** [comment]
Same issue, AT site: 34252724-hc and same solution! 🤸

**Comment by allilevine** [comment]
Hi @ramonjd @Automattic/create 👋 Can you help us understand if this issue is related to Coming Soon (public by default) v1/v2? PCYsg-u4S-p2 It's happening on both Simple and Atomic sites. All of the affected sites have switched from a classic theme to a block theme at some point. See p1645737497278649/1645694640.072909-slack-CKZHG0QCR for examples.

cc @Robertght

**Comment by ramonjd** [comment]
Thanks for reporting everyone! @allilevine I'm taking a quick look this morning, but might not be as fast as @Automattic/ganon, who I think has had more recent eyes on the relevant code.

**Comment by ramonjd** [comment]
I'm having trouble reproducing the problem. 

I've tried the launch flow on a private and AT site (`/start/launch-site/*`) a few times. Both switch from coming soon to public. 

Toggling between Coming Soon, public and private appears to work as I expected. Changing from classic to block themes didn't affect anything in my tests.

I've had a look at the history of a couple of the blogs that were having trouble and can't pinpoint a common criterion. It would be great to confirm that. 

One thing I noticed was that sometimes the site didn't reflect changes to the `launch-status` option. So, when I switched to coming soon mode and manually set the option to `unlaunched` I'd have expected, after a short moment, to see the launch status presented to me. I can't, however, find anywhere that might suggest this would be blocking sites from going public.

If there are further reports, it would be helpful (if possible) to describe the steps the user took to get to the problem, especially if they are simple sites (which are easier to debug and check the options). 

I've tried both fresh sites and existing. 

cc @roo2

**Comment by roo2** [comment]
I also haven't been able to reproduce it yet. I looked for comonalities between sites, these things were NOT common
FSE vs legacy theme
Simple vs Atomic
Changes to domain name
"Old" sites vs new site

The only thing I've noticed so far is that they all register the blog-privacy changing to public before the launch-site event. This happens when the site is set to public via the settings page, but not when the blog is launched via the launch site flow from the editor or the my-home page. There is some code which intercepts blog privacy changing (the `blog_public` site option) and launches the site, maybe the issue is in some way related to this code 🤔 

+1 for if you find a user where this is happening it would be useful to know how the user attempted to launch the site previously. It would also be very useful to know the value of the following site options before fixing the site:
`wpcom_public_coming_soon`
`blog_public`
`launch-status`
This will help to narrow down how this situation is occuring.

**Comment by poeticabdul** [comment]
Reported here 34283859-hc

**Comment by Robertght** [comment]
Before applying the workaround please check this Slack thread p1646064764096039-slack-CB0B2G43X 

### **Later update: Please apply the workaround as before.**

**Comment by JoshuaGoode** [comment]
This can occur on both Simple and Atomic but is much more common on Atomic.

It's easy for things to get out of sync on Atomic.

It can also be easily reproduced by migrating in a self-hosted site to a fresh Atomic site -- as the migration process doesn't adjust the options.

Here's the common flow for instances where a user is migrating in a site via our tool or a third-party plugin.

1. Create a new Atomic site.
2. Migrating in a self-hosted or local site.
3. `Wpcom Public Coming Soon` will often be left as 1 on the WPCOM side but the Atomic site likely doesn't have the `full-site-editing` (WPCOM Editing Toolkit) plugin active. 


On normal sites, where a migration didn't occur, it is normally due to the `Wpcom Public Coming Soon` option being incorrect for some reason on the WPCOM side. @roo2

**Comment by sguzovskaia** [comment]
New report zen-4820328

**Comment by ramonjd** [comment]
> Atomic site likely doesn't have the full-site-editing (WPCOM Editing Toolkit) plugin active.

It's been a while, but as far as I can remember if the Editing Toolkit isn't active or installed, the Coming Soon page and related functionality won't show or work at all. @roo2 might be able to confirm.

As for Simple Sites, Coming Soon mode means `wpcom_public_coming_soon === 1 && blog_public === 0`. All I can think of... thinking back 12 months now 😄 is:

- launch isn't setting the options correctly (to `blog_public === 1` and `wpcom_public_coming_soon === 0`)
- toggling visibility settings away from "Coming Soon mode" isn't correctly setting the options
- ?

I couldn't reproduce any of these unfortunately. 🤔

**Comment by roo2** [comment]
the affected site that @Robertght showed me was an atomic site that had 
```
wpcom_public_coming_soon=1  # (this should have been set to 0!)
blog_public=1
launch-status=launched
```

I'll continue looking into this, I'll look into how this might happen and how we can either prevent this from happening in the future, or handle scenarios where this does happen. One option might be to only treat a blog as being in "coming soon" mood if `blog_public` is `0`.

**Comment by allilevine** [comment]
> I'll continue looking into this, I'll look into how this might happen and how we can either prevent this from happening in the future, or handle scenarios where this does happen. One option might be to only treat a blog as being in "coming soon" mood if `blog_public` is `0`.

@roo2 Thank you for taking this on! 🙌

**Comment by roo2** [comment]
tl;dr; maybe the ETK was disabled (or not loaded?) and the user updated the setting via the wp-admin page (which should be disabled?). I'll try adding some code to the wpcom-site-helper which would catch this case.

hmm i thought maybe disabling the editing toolkit and toggling the coming soon mode would cause this, but that seemed to make the site public as expected. 

I did notice that on an atomic site, if I set `blog_public` to 1 from the CLI, `wpcom_public_coming_soon` is still `0` (which is the state we saw with this site). We disable the wp-admin section which would allow a user to set `blog_public` directly on atomic sites, so it _shouldn't_ be happening. 

We currently have two hook that update `wpcom_public_coming_soon` if `blog_public` is updated. One [in the ETK](https://github.com/Automattic/wp-calypso/blob/trunk/apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon.php#L103), and a similar hook in the public API which is triggered when an atomic site uses the calypso settings page `/settings/general/$site.wpcomstaging.com`. But no hook for an atomic site being updated via wp-admin (which _shouldn't_ be possible) when the ETK is disabled.

I might try to add a hook to wpcom-site-helper to ensure it's run if `blog_public` is changed on the remote site for any reason, `wpcom_public_coming_soon` is set correctly and the site is launched.

I'll post here once I get it deployed.

Assuming that does fix the root cause, there might be some sites that are already in this state that will still need to be fixed with the same workaround.

**Comment by roo2** [comment]
These are notes to myself and @ramonjd or anyone else who understands the arcane magic of "coming soon":

It was a little different than I remembered. We use Atomic Persistent data for `wpcom_public_coming_soon` on the atomic site. We actually override the getter for `wpcom_public_coming_soon` in wpcomsh to return the Atomic Persistent data.

This means that we can't intercept setting `blog_public` and change the remote site's `wpcom_public_coming_soon` like we do on simple sites. 

We currently rely on changes to `blog_public` being done via the API (from the calypso settings page) If the change is done directly or via wp-admin on the _remote_ atomic site, then we get into trouble, whether the ETK is active or not.

I am a bit confused because it looks like changes to `blog_public` should be synced to the cloud site. This should then allow the code in wpcom to respond to that option change and set wpcom_public_coming_soon, which is then synced back to the atomic site's Atomic Persistent data.

The issue is that changes to blog_public aren't being jetpack synced. They seem to be whitelisted, it looks like they _should_ be, but they're not. 

The investigation continues...

**Comment by roo2** [comment]
I created a PR in wpcomsh to allow syncing of `blog_public`, this should cause the site to be properly launched if for any reason blog_public gets set to 1 on the remote site 831-gh-Automattic/wpcomsh

cc @ramonjd I think you're the only other code wrangler who's across this feature at all 😂 😭

**Comment by nagpai** [comment]
34691211-hc  Simple site. Worked after turning it off and on.

**Comment by dcoleonline** [comment]
another in 34703576-hc

**Comment by Robertght** [comment]
4863608-zen

**Comment by cintacks** [comment]
Another one. Turning it off and on again always works: 4881433-zd-woothemes

**Comment by as-esu** [comment]
4899281-zen simple site

**Comment by kylemcph** [comment]
User in 4913073-hc was waiting over a week for two sites to be made public before reaching out to support so I could perform the "Privacy somersault".

**Comment by Greatdane** [comment]
another: 4892067-zd-woothemes (turned on and off again)

**Comment by Robertght** [comment]
4906490-zen

**Comment by valterlorran** [comment]
Another: 4910318-zen

**Comment by p-jackson** [comment]
I've just read through the blogrc audit log for many of these sites. Something they _all_ seem to have in common, both simple and atomic, is that they change their primary domain _before_ launching. Whether that's through a domain purchase or by changing to a `.wpcomstaging.com` domain as part of AT. Either way it's a change of primary domain.

I still haven't reproduced it myself, but I'm wondering whether that's a clue into what's going wrong.

**Comment by simison** [comment]
I first thought this could be related to some Jetpack sync/Atomic issues (p9F6qB-90M-p2 or Slack p1648110864091759-slack-jetpack-developers), but it shouldn't have anything to do with Jetpack options sync since values are stored in Atomic persistent data, right? Do those persistent data tables use domains as a key in any way?

**Comment by dragstor** [comment]
35010513-HC

**Comment by p-jackson** [comment]
> it shouldn't have anything to do with Jetpack options sync since values are stored in Atomic persistent data, right?

@simison The setting is only moved to persistent data when the site is atomic, otherwise it's just a regular old site option. The site option is intentionally not sync'd by Jetpack though since it's supposed to use persistent data. There could be issues with this approach, which is what's being investigated in 831-gh-Automattic/wpcomsh. But given that this also happens on simple sites I'm not sure Jetpack is the root cause.

> Do those persistent data tables use domains as a key in any way?

This sparked an idea! When calling the site launch endpoint from the launch flow, we use the site slug in the URL i.e. `/site/:siteSlug/launch`. So if it's using an old site slug then it could be causing the problem.

There's also some suspicious stuff going on in the backend. @roo2 do you mind reviewing D78083-code which adds some simple logging that can prove/disprove any suspicion?

**Comment by zyousafi** [comment]
Same issue reported here: #34478831-hc

The suggested workaround (i.e., toggle from Public to Private, and back to Public) worked for the user 🙂

**Comment by Greatdane** [comment]
Another here, this is the first time I have seen it on a Simple site; 4926719-zd-woothemes

**Comment by nagpai** [comment]
#35074234-hc

**Comment by kylemcph** [comment]
#13817999-hc. "Privacy somersault" worked.

**Comment by kylemcph** [comment]
#4935503-zen. "Privacy somersault" worked.

**Comment by igotdes** [comment]
Also reported in #4936779-zen (simple site). The workaround of setting to private and back to public helped.

**Comment by kylemcph** [comment]
#34876815-hc. "Privacy somersault" worked.

**Comment by pauljacobson** [comment]
I received a report of this issue too, in 4947106-zen

In this case, I was able to resolve the issue by enabling the _Discourage search engines from indexing this site_ option, saving the settings, and then deselecting that option (and saving the site settings).

![Capture_2022-04-14 at 09 41 32](https://user-images.githubusercontent.com/2912202/163329325-a9147607-1c00-4f20-a519-2ee762e0e820.jpeg)

This particular site is on a WordPress.com Personal Plan. Looking at the site's audit trail, it doesn't look like the site registered the user's initial switch to _Public_, or mine when I was testing this. It only registered the workaround.

**Comment by ariel-maidana** [comment]
Another report: 35161105-hc

Setting it to private and then public again worked.

**Comment by roo2** [comment]
@p-jackson it looks like we are seeing cases where the blog_id isn't matching on launch 🎉 2c352-pb

**Comment by tellisbethel** [comment]
Another report here: 34900628-hc
Toggled between Coming Soon and Public.

**Comment by p-jackson** [comment]
Thanks @roo2 I only just checked this! Good to see that hypothesis came to something! Next up I'll make a diff that consistently uses the same method during launch: either always use the blog id parameter, or always use the "current blog". Rather than using a mixture of both ways to apply the change to the site.

**Comment by cuemarie** [comment]
Same issue here: 35068705-hc
Privacy Somersault to resolve

**Comment by csabarakasz** [comment]
5184959-zd-woothemes Site was Public but live it showed coming soon. Switched to Private (there was no Coming soon option) and Public again, and that made the site Public. 

Pro plan. AT site. Byrne theme

**Comment by zdenys** [comment]
5186653-zd-woothemes

**Comment by ariel-maidana** [comment]
29365431-hc
Simple site.
Setting privacy to Coming Soon and then to Public again worked.

**Comment by st8matrix** [comment]
5203051-zen

**Comment by brittanygray902** [comment]
Found a similar case here: 35523522-hc

I was able to solve by setting site to "Coming Soon" and then setting back to "Public"

**Comment by tellisbethel** [comment]
Another case here: 5233808-zd-woothemes - 
Pro plan site was set to Public but live it showed Coming Soon. Asked user to switch to Private (there was no Coming soon option) and Public again.

**Comment by p-jackson** [comment]
I think GitHub just auto-closed this issue, so it could be a bit pre-emptive. But hopefully (🤞🤞🤞🤞) #63947 will fix the bug (thanks @roo2 🎉)
If it does continue to get reported by customers then we'll need to reopen.

**Comment by roo2** [comment]
Yea I just saw, I think we'll leave it open for a little bit just so people can find it if this does keep happening.

Fingers crossed https://github.com/Automattic/wp-calypso/pull/63947 fixes the issue for everyone though 🤞

**Comment by metabreakr** [comment]
19251354-hc

**Comment by poeticabdul** [comment]
Reported here 35171142-hc

**Comment by p-jackson** [comment]
🤔 
The audit log for 19251354-hc looks like it might be a different issue. There is no record of the launch event happening prior to them reaching out to support. So it could be they never completed the flow or the launch just failed in some other way.

The 35171142-hc one looks more like this issue though. It's Atomic while the issue above is Simple. It'll be interesting if that's a pattern that emerges if more reports continue to come in.

**Comment by metabreakr** [comment]
Hmm, for 19251354-hc the site was definitely set to public. 

I have another user in 35282172-hc as well.

**Comment by cuemarie** [comment]
Encountered this here 33083930-hc after fixing a JP connection issue. 

Once JP was reconnected, site showed `Coming Soon`, but Settings page showed `Private` - meanwhile the site had been set to `Public` 2 years ago and never changed again.

I activated the WPcom Editing Toolkit plugin, and then the Settings Page showed `Public`, but the `Coming Soon` sticker remained. From here, I was not able to swap the settings to Coming Soon - with each attempt, the selection would pop back to Public. 

Thanks to @pauljacobson 's comment, I was able to resolve by saving the site as `Public NoGoogle`, then returning to regular old `Public`. 

Way more somersaults than usual! 🤸 🤸 🤸

**Comment by Robertght** [comment]
cc @p-jackson and @roo2 on the above details in case it helps us find a permanent fix.

**Comment by nagpai** [comment]
35994310-hc

~Did the privacy settings back and forth..well.. somersault .. worked fine after that.~

It was cloudflare! Sorry.

**Comment by roo2** [comment]
I see that 35171142-hc actually occurred before the fix went in (the user just took a while to notice!), same for 35282172-hc. It looks like the other issues where related to jetpack/cloudflare/the ETK being uninstalled.

As far as I can tell, the original issue was indeed fixed by https://github.com/Automattic/wp-calypso/pull/63947 🎉 

I'll close this issue, I think the other issues can just be discussed as normal in #wpcom-support, I get a ping when anyone mentions "Coming Soon"

**Comment by JoshuaGoode** [comment]
Hi there @roo2 

Does https://github.com/Automattic/wp-calypso/pull/63947 consider and work with Atomic sites as well?

**Comment by cuemarie** [comment]
I encountered this issue again today here: 36389445-hc

Site was saved as Public under Settings, but showing the 'Coming Soon' landing page. Saving as 'Coming Soon', then 'Public', resolved the issue as usual. 

Based on @roo2 's comments above, I'm guessing this site may have also just been this way since before the issue was resolved, but I wanted to note it here just in case. Will leave the issue closed

**Comment by cuemarie** [comment]
Another one today: 36404972-hc

**WordPress.com Editing Toolkit** plugin was inactive, so `Coming Soon` wasn't an option under the Privacy section, but the site was still showing the `Coming Soon` sticker. 

Activated the plugin, ran privacy somersault, and deactivated again to resolve.

**Comment by ariel-maidana** [comment]
I saw a similar case here: 37608501-hc
The site has many plugins, so it could be a conflict. I'm leaving this comment here in case other instances come up and we need to reopen this issue.
